movies/domain/src/main/java/com/example/movies/domain/repositories/MoviesRepository.kt

```diff
@@ -3,16 +3,16 @@ package com.example.movies.domain.repositories
 import com.example.kotlinhelpers.Response
 import com.example.movies.domain.entities.Movie
 import com.example.movies.domain.entities.MovieDetail
-import kotlinx.coroutines.flow.Flow
 import com.example.movies.domain.entities.MovieList
+import com.example.movies.domain.entities.MovieRecommendationList
+import kotlinx.coroutines.flow.Flow
 
 interface MoviesRepository {
     suspend fun getListMovies(page: Int?): Response<MovieList>
     suspend fun getDetailMovie(id: Int): Response<MovieDetail>
-
+    suspend fun getRecommendedMovies(id: Int): Response<MovieRecommendationList>
```

Comment:
change the interface to can implement the repository with the page in recomendations.